### ISSUE TITLE
Upgrade PRAW dependency

### ISSUE THREAD

**Issue by TheLonelyGhost** [GrafeasGroup/tor#173]
Currently we're on v5.0.0 which is horrendously out-of-date. We need to upgrade this to the latest juicy goodness so some bugs can hopefully be squashed for free.

**Comment by NGTmeaty** [comment]
I wonder if #156 would be a better way to go than updating PRAW?

**Comment by TheLonelyGhost** [comment]
Agreed, but it's something that I haven't had time to work on in quite a while so I'm opting for the quick win for now. Would definitely like to give snooworld a go. :)

I think a change we have coming in the pipeline which isolates libraries used by each bot instead of having them in the same soup will be of benefit here too. This means the task of upgrading to the latest compatible versions of libraries becomes easier to test and to revert.

**Comment by NGTmeaty** [comment]
Yeah, just tried to give it a shot here, but ran into a few issues with actually testing if the new PRAW version worked. The changelogs don't seem to have any breaking issues with the functions we use though?

**Comment by TheLonelyGhost** [comment]
Yeah, short of reading through the implementation of the PRAW code + reading through the API calls (filtering to the methods we actually use in the bots), there's no good way of testing this besides just running it and seeing what happens. :/

One of the reasons I wanted to add mypy support for it. At least it would have type safety then.
